accounts/CHANGELOG.md

```diff
@@ -0,0 +1,444 @@
+# Changelog for [unreleased] (UNRELEASED)
```

Comment:
Codacy found an issue: [[no-shortcut-reference-link] Use the trailing `[]` on reference links](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=6202733)